apps/accounts/views.py

```
@@ -5,11 +5,11 @@
 from rest_framework.permissions import AllowAny
 from rest_framework.response import Response
 from rest_framework.views import APIView
-from drf_spectacular.utils import extend_schema, OpenApiResponse
+from drf_spectacular.utils import extend_schema, OpenApiResponse, OpenApiExample
```

Ruff

`drf_spectacular.utils.OpenApiExample` imported but unused

Remove unused import: `drf_spectacular.utils.OpenApiExample`

(F401)